SPECS/vim/vim.spec

```diff
@@ -11,6 +11,7 @@ URL:            https://www.vim.org
 Source0:        https://github.com/%{name}/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 BuildRequires:  ncurses-devel
 BuildRequires:  python3-devel
+Requires:       sed
```

Comment:
If it's required by the post-install script, then please user `Requires(post)`. Otherwise, `sed` might be installed after `vim` and the script will still fail.